community/modules/compute/pbspro-execution/variables.tf

```diff
@@ -0,0 +1,314 @@
+    project = "cloud-hpc-image-public"
+  }
+}
+
+variable "disk_size_gb" {
+  description = "Size of disk for instances."
+  type        = number
+  default     = 200
+}
+
+variable "disk_type" {
+  description = "Disk type for instances."
+  type        = string
+  default     = "pd-standard"
+}
+
+variable "auto_delete_boot_disk" {
+  description = "Controls if boot disk should be auto-deleted when instance is deleted."
+  type        = bool
+  default     = true
+}
+
+variable "local_ssd_count" {
+  description = "The number of local SSDs to attach to each VM. See https://cloud.google.com/compute/docs/disks/local-ssd."
+  type        = number
+  default     = 0
+}
+
+variable "local_ssd_interface" {
+  description = "Interface to be used with local SSDs. Can be either 'NVME' or 'SCSI'. No effect unless `local_ssd_count` is also set."
+  type        = string
+  default     = "NVME"
+}
+
+variable "enable_public_ips" {
+  description = "If set to true, instances will have public IPs on the internet."
+  type        = bool
+  default     = true
+}
+
+variable "machine_type" {
+  description = "Machine type to use for the instance creation"
+  type        = string
+  default     = "c2-standard-60"
+}
+
+variable "labels" {
+  description = "Labels to add to the instances. List key, value pairs."
+  type        = any
+}
+
+variable "service_account" {
+  description = "Service account to attach to the instance. See https://www.terraform.io/docs/providers/google/r/compute_instance_template.html#service_account."
+  type = object({
+    email  = string,
+    scopes = set(string)
+  })
+  default = {
+    email = null
+    scopes = [
+      "https://www.googleapis.com/auth/devstorage.read_write",
+      "https://www.googleapis.com/auth/logging.write",
+      "https://www.googleapis.com/auth/monitoring.write",
+      "https://www.googleapis.com/auth/servicecontrol",
+      "https://www.googleapis.com/auth/service.management.readonly",
+      "https://www.googleapis.com/auth/trace.append",
+    ]
+  }
+}
+
+variable "network_self_link" {
+  description = "The self link of the network to attach the VM."
+  type        = string
+  default     = "default"
+}
+
+variable "subnetwork_self_link" {
+  description = "The self link of the subnetwork to attach the VM."
+  type        = string
+  default     = null
+}
+
+variable "network_interfaces" {
+  description = <<-EOT
+    A list of network interfaces. The options match that of the terraform
+    network_interface block of google_compute_instance. For descriptions of the
+    subfields or more information see the documentation:
+    https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance#nested_network_interface
+
+    **_NOTE:_** If `network_interfaces` are set, `network_self_link` and
+    `subnetwork_self_link` will be ignored, even if they are provided through
+    the `use` field. `bandwidth_tier` and `disable_public_ips` also do not apply
```

Comment:
nit: `disable_public_ips` should be `enable`

Reply:
Resolved on HEAD of PR